operator/templates/backup-job.yaml

```diff
@@ -0,0 +1,22 @@
+{{- $root := . -}}
```

Comment:
Nit: Not needed, the root is always accessibly with `$`. E.g. ``$.Name`. See `$.Params.KUBECTL_VERSION` used in this file.